tests-config.json

```diff
@@ -0,0 +1,2703 @@
+
+type Test = {
+    testFile?: string;
+    isWs?: boolean;
+    public?: boolean;
+    args?: any[];
+    skippedProperties?: {}
+    skip?: string;
+}
+
+type Tests = Test & {
+    skip?: string;
+    skipWs?: string;
+    skipPhpAsync?: string;
+    until?: string;
+    httpProxy?: string;
+    httpsProxy?: string;
+    wsProxy?: string;
+    wssProxy?: string;
+    timeout?: number;
+}
+
+type TestConfig = {
+    [key: string]: Tests;
+}
+
+// *********************************
+// ***** AUTO-TRANSPILER-START *****
+function config (symbol: string = 'BTC/USDT', code: string = 'USDT', isSpot: boolean = true) {
+    return ({
+        'exchange': {
+            // public tests
+            'loadMarkets': {
+                'testFile': 'loadMarkets',
+                'args': [],
+                'isWs': false,
+                'public': true,
```

Comment:
actually, `public/private` flags should be deprecated in my opinion. i'll explain it later.